docs/cli-reference.md

```diff
@@ -72,35 +72,40 @@ Options:
 
       --network <NAME>
           Network that the consensus and execution nodes are operating on
-          
+
           [default: mainnet]
 
       --new-payload-wait-millis <MILLIS>
           Maximum time that a consensus node should wait for a newPayload response from the cache.
-          
+
           We expect that the controlling consensus node and primary execution node will take some time to process requests, and that requests from consensus nodes could arrive while this processing is on-going. Using a timeout of 0 will often result in a SYNCING response, which will put the consensus node into optimistic sync. Using a longer timeout will allow the definitive (VALID) response from the execution engine to be returned, more closely matching the behaviour of a full execution engine.
-          
+
           [default: 2000]
 
       --new-payload-wait-cutoff <NUM_BLOCKS>
           Maximum age of a payload that will trigger a wait on `newPayload`
-          
+
           Payloads older than this age receive an instant SYNCING response. See docs for `--new-payload-wait-millis` for the purpose of this wait.
-          
+
           [default: 64]
 
       --fcu-wait-millis <MILLIS>
           Maximum time that a consensus node should wait for a forkchoiceUpdated response from the cache.
-          
+
           See the docs for `--new-payload-wait-millis` for the purpose of this timeout.
-          
+
           [default: 1000]
 
       --body-limit-mb <MEGABYTES>
           Maximum size of JSON-RPC message to accept from any connected consensus node
-          
+
           [default: 128]
 
+      --ee_timeout_millis <MILLIS>
```

Comment:
should be `--ee-timeout-millis`

did you generate this text from the binary?

Reply:
no, i wrote it by myself.